src/dotnet-aspnet-codegenerator/DotNetBuildCommandHelper.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.DotNet.Cli.Utils;
+using NuGet.Frameworks;
+
+namespace Microsoft.Extensions.CodeGeneration
+{
+    internal class DotNetBuildCommandHelper
+    {
+        internal static int Build(
+            string project,
+            string configuration,
+            NuGetFramework framework,
+            string buildBasePath)
+        {
+            // TODO: Specify --runtime?
+            var args = new List<string>()
+            {
+                project,
+                "--configuration", configuration,
+                "--framework", framework.GetShortFolderName(),
+            };
+
+            if (buildBasePath != null)
```

Comment:
Should do what's done [here](https://github.com/aspnet/Common/blob/cbcc732a7e9bc51debdf4c5db5ad19de4ba64d75/src/Microsoft.Extensions.DotnetToolDispatcher.Sources/DotnetToolDispatcher.cs#L47-L51).